### ISSUE TITLE
[Bug][a11y] Should be able to select separately the number of selected tabs to be added to a collection

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#16426]
## Prerequisites
Talkback should be enabled
Have a couple couple of open tabs

## Steps to reproduce
1. Tap the "Save to collection" button
2. Select a couple of tabs
3. Tap on the number of selected tabs from the action bar 

### Expected behavior
Should be able to select only the number of selected tabs item

### Actual behavior
Tapping the number of selected tabs highlights the entire action bar 

### Device information

* Android device: 
• Google Pixel 3a (Android 11)
• Huawei Mate 20 Lite (Android 10)
• OnePlus A3 (Android 6.0.1)

* Fenix version: Nightly Build 201106
 
### Notes
**Reproducible in both List & Grid view**

► Screenshot
![rsz_screenshot_20201106-164007](https://user-images.githubusercontent.com/51314259/98380151-dbadb900-2050-11eb-9bef-0aedc42edb56.png)

**Comment by AndiAJ** [comment]
Hi, I've just checked this matter on the latest **`Nightly Build 201110`** using the following devices:
• Google Pixel 3a (Android 11)
• Huawei Mate 20 Lite (Android 10)
• OnePlus A3 (Android 6.0.1)

It works properly on:
✔️ Google Pixel 3a (Android 11)
✔️ Huawei Mate 20 Lite (Android 10)

On the other hand, I **can't select** it on the OnePlus A3 (Android 6.0.1)
• Tapping the the number of selected tabs actually selects the entire view
• It gets skipped when I try to reach it with the swipe gesture 

► Video
![20201110-095523](https://user-images.githubusercontent.com/51314259/98646358-ef506c80-233b-11eb-9e0b-5e20869b2703.gif)

**Comment by mcarare** [comment]
Thank you @AndiAJ for testing!
If QA can test this on another device with Android 6 and it works as expected this can be closed.
Then, for the OnePlus A3, a separate issue should be opened with other a11y issues specific to the device (like https://github.com/mozilla-mobile/fenix/issues/11879).

**Comment by AndiAJ** [comment]
Hi @mcarare , It seems that only the OnePlus A3 (Android 6.0.1) misbehaves.
I've just opened a separate issue for this matter #16474 

It works properly on:
✔️ Google Pixel 3a (Android 11)
✔️ Huawei Mate 20 Lite (Android 10)
✔️ OnePlus 6T (Android 9)
✔️ Nexus 5 (Android 6.0.1)
